storage/operation/node_disallow_list.go

```diff
@@ -0,0 +1,39 @@
+package operation
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/storage"
+)
+
+// PurgeNodeDisallowList removes the set of disallowed nodes IDs from the database.
+// If no corresponding entry exists, this function is a no-op.
+// No errors are expected during normal operations.
+//
+// TODO: TEMPORARY manual override for adding node IDs to list of ejected nodes, applies to networking layer only
+func PurgeNodeDisallowList(w storage.Writer) error {
+	err := RemoveByKey(w, MakePrefix(blockedNodeIDs))
+	if err != nil && !errors.Is(err, storage.ErrNotFound) {
```

Comment:
RemoveByKey will always succeed regardless the key exists or not. In other words, it never return storage.ErrNotFound. See comments for `Writer.Delete`:
>	// Delete deletes the value for the given key. Deletes are blind all will
	// succeed even if the given key does not exist.

```suggestion
	if err != nil  {
```